### PR TITLE
Fix feature tests

### PR DESCRIPTION
**Pull request by chao-xian**
A change in a previous commit[1] was not picked up by the branch build
for some reason, and only failed once merged into master. This PR should
fix the issue.

[1] https://github.com/alphagov/whitehall/pull/4904

### CODE REVIEW

File: features/step_definitions/social_media_steps.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@
     social_container = Organisation.last
     visit organisation_path(social_container)
   end
-  assert page.has_css?(".social-media-accounts .social-media-link.#{social_service.parameterize}", text: "Connect with #{social_container.display_name} on #{social_service}")
+  assert page.has_css?(".social-media-accounts .social-media-link.#{social_service.parameterize}", text: "#{social_service}")
```
Comment (injms) [inline features/step_definitions/social_media_steps.rb]: Picky, and just realised this:

```suggestion
  assert page.has_css?(".social-media-accounts .social-media-link.#{social_service.parameterize}", text: social_service)
```

File: features/step_definitions/social_media_steps.rb [hunk 2]
```diff
@@ -48,5 +48,5 @@
     social_container = Organisation.last
     visit organisation_path(social_container)
   end
-  assert page.has_css?(".social-media-accounts .social-media-link.#{social_service.parameterize}", text: "Connect with #{social_container.display_name} on #{title}")
+  assert page.has_css?(".social-media-accounts .social-media-link.#{social_service.parameterize}", text: "#{title}")
```
Comment (injms) [inline features/step_definitions/social_media_steps.rb]: Similar to previous:

```suggestion
  assert page.has_css?(".social-media-accounts .social-media-link.#{social_service.parameterize}", text: title)
```

**Comment by emmabeynon** [comment]
Just an FYI this probably happened because cucumber tests hadn't been running for a while.  Fixed in this https://github.com/alphagov/whitehall/pull/4912, which was merged yesterday.

**Comment by chao-xian** [comment]
[(2) Mark up share links as English](https://trello.com/c/sET78e3z/14-2-mark-up-share-links-as-english)

**Pull request merged**